src/main/scala/sigmastate/eval/RuntimeCosting.scala

```diff
@@ -1692,12 +1692,12 @@ trait RuntimeCosting extends CostingRules with DataCosting with Slicing { IR: Ev
         val c = opCost(Seq(lC.cost, rC.cost), costOf(node))
         withConstantSize(v, c)
 
-//      case BinXor(l, r) =>
-//        val lC = evalNode(ctx, env, l)
-//        val rC = RCostedThunk(Thunk(evalNode(ctx, env, r)), 0)
-//        val v = sigmaDslBuilder.binXor(lC.value, rC.value)
-//        val c = lC.cost + rC.cost + costOf(node)
-//        withDefaultSize(v, c)
+      case BinXor(l, r) =>
+        val lC = evalNode(ctx, env, l)
+        val rC = RCostedThunk(Thunk(evalNode(ctx, env, r)), 0)
+        val v = BinaryXorOp.applyLazy(lC.value, rC.value)
+        val c = opCost(Seq(lC.cost, rC.cost), costOf(node))
```

Comment:
Please use opCost(v, Seq(lC.cost, rC.cost), costOf(node)) version
Please see this comment https://github.com/ScorexFoundation/sigmastate-interpreter/pull/477#discussion_r278424120

Reply:
This change leads to fail
```
scalan.TypeDescs$$anon$7 cannot be cast to special.collection.impl.CostsDefs$Costed$CostedElem
java.lang.ClassCastException: scalan.TypeDescs$$anon$7 cannot be cast to special.collection.impl.CostsDefs$Costed$CostedElem
```